### ISSUE TITLE
(UX inconsistency) Generate New Address behaviour

### ISSUE THREAD

**Issue by gravityz3r0** [planetdecred/dcrandroid#483]
Regarding latest v1.5.0 wallet.

Found inconsistencies issue with the receive tab:

1. By default, the receiving address does not change when navigating in and out of "Receive DCR" windows. (None of the address is used)
2. Click the 3 dots on the right top corner to "generate new address"
3. Address changes instantly. 
4. Now, navigate out of "Receive DCR" windows and re-display. Address changes again. (Ie: address advances the second time)
5. Any further navigation in and out of "Receive DCR windows" will not change the address anymore. (Until you generate a new address again)

To sum things up: Each click of "generate new address" causes new address to be "advanced" twice consecutively instead of once.

**Comment by dreacot** [comment]
I can confirm this still happens, moving to To-do
